integreat_cms/cms/templatetags/content_filters.py

```diff
@@ -96,6 +96,28 @@ def remove(queryset, instance):
     return queryset.exclude(id=instance.id)
 
 
+@register.filter
+def sort_languages(other_languages, current_language):
+    """
+    This filter sorts languages in language tabs
+
+    :param current_language: The current language
+    :type current_language: string
+
+    :param other_languages: Other languages
+    :type other_languages: list
+
+    :return: the filtered list with the current language on the second position
+    :rtype: list
+    """
+    if other_languages[0] != current_language:
+        other_languages = list(other_languages)
+        other_languages.remove(current_language)
+        other_languages.insert(1, current_language)
+
+        return other_languages
```

Comment:
```suggestion
    return other_languages
```
If you only return the languages in the `if` case, the language tabs will be invisible on the default language